### DISCUSSION TITLE
Item categories

### DISCUSSION THREAD

**Discussion by venkatramanm** [beckn/protocol-specifications#175] [Open Commerce Networks]
Instead of defining these categories by each BPP, we can use the HSN/sac codes

That way, every party will have a common language to talk and query by item category

**Comment by Vedsaga** [comment]
This will surely going to helpful to optimizing the serach operation for BPPs and could be helpful while implementing #127

So, idea could be BAP include HNS code in the search message and then when BPPs will receive the request it first try to search in it's server using HNS code and filter out the details and then again BPPs can use name to segregate the results and then forward the results via on_serach

**Comment by ravi-prakash-v** [comment]
Is this an international standard or only for India?

**Reply by Vedsaga** [reply]
I was researching and it is Global Standard followed by countries who are members of World Customs Organization (WCO). Now, HSN code have different classification some countries may uses `6 digit` and `8 digit ` code based on nature of goods and services.  HSN is must for all the manufacturers & services companies to tag there goods & services because government have classified the GST slabs based on HSN code. I believe we shall approach some one who is expert that would help us understand the up and down side of this. Because, as far as implementation is concerned implementing this will surely going create more standardized systems in beckn and will be helpful in future if new kind of network is build based on Beckn network who offers taxation services / analytical services because currently a goods (A biscuit packet ) can be named differently based on language & region that retailers operate in however to enable global connectivity there is need for certain standardization. And, implemention of HSN will be first step in that direction then later we can have discussion on assignment of w `Global  Unique Tag` to each goods I saw dicussion is already open on this topic #178...
